components/roadmap-grid/grid-row.tsx

```diff
@@ -33,8 +35,9 @@ export function GridRow({
   const closestDateIdx = Math.round(timeScaler.getColumn(dayjs.utc(milestone.due_date.get()).toDate()));
   const span = Math.max(4, numGridCols / timelineTicks.length);
   const closest = span * (closestDateIdx - 1);
+  const viewMode = useViewMode();
 
-  const childLink = getLinkForRoadmapChild({ issueData: milestone.get(), query: useRouter().query, currentRoadmapRoot: issueDataState.value });
+  const childLink = getLinkForRoadmapChild({ viewMode, issueData: milestone.get(), query: useRouter().query, currentRoadmapRoot: issueDataState.value });
```

Comment:
getLinkForRoadmapChild should use viewMode so children clickthroughs are using the same view. we could enhance that logic, but as a basic rule, I think that's goodenough UX